src/nerve/frontends/tui/commander/workflow_runner.py

```
@@ -0,0 +1,1028 @@
+
+    async def _poll_workflow(self) -> None:
+        """Poll workflow status and update state."""
+        last_event_count = 0
+
+        while self.state not in ("completed", "failed", "cancelled"):
+            # Check for cancellation request
+            if self.cancelled:
+                try:
+                    await self.adapter.cancel_workflow(self.run_id)
+                    self._add_event("workflow_cancelled", {"by": "user"})
+                except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)